modules/draco/src/lib/draco-module-loader.ts

```diff
@@ -59,14 +63,28 @@ async function loadDracoDecoder(options) {
   let wasmBinary;
   switch (options.draco && options.draco.decoderType) {
     case 'js':
-      DracoDecoderModule = await loadLibrary(DRACO_JS_DECODER_URL, 'draco', options);
+      DracoDecoderModule = await loadLibrary(
```

Comment:
This seems to duplicate functionality in` loadLibrary`, which also looks at options IIRC.

In that case I think we should consider:
- seeing if we can leverage / extend the loadLibrary function to handle this.
- strip out that functionality from loadLibrary, so that there is not two places that handle parts of this logic.

Reply:
Added `useLocalLibraries` option to update `loadLibrary` logic.